package/MDAnalysis/topology/tpr/utils.py

```diff
@@ -46,6 +46,8 @@
 
 The module also contains the :func:`do_inputrec` to read the TPR header with.
 """
+import warnings
+
```

Comment:
I don't think `warnings` is needed for the PR against develop. In 2.0.0 we will index TPR resids from 1 by default.